src/app/shared/store/nodes-sorting/columns.data.ts

```diff
@@ -60,6 +60,19 @@ export const columns: Column[] = [
     show: value => value || '',
     link: (value, {node}) => value && `${environment.blockscoutUrl}/address/${node.id.toString()}/transactions`,
   },
+  {
+    name: 'Version',
+    icon: 'code',
+    variants: ['xlarge'],
+    accessor: (node) => {
+      if (node.info?.node) {
+        const parts = node.info?.node.split('/')
+        if (parts.length > 0) {
+          return parts[1]
```

Comment:
You can use the `?` on the array part too like: `node.info?.node.split('/')?.[1]`

Reply:
changed, thanks